homeassistant/backports/__init__.py

```diff
@@ -1,4 +1,4 @@
-"""Enum related utilities."""
+"""Backports from newer Python versions."""
```

Comment:
Let's move it to match the Python standard lib structure, but under `homeassistant.backports`. So change this file to `homeassistant/backports/enum.py`

Reply:
Makes sense, I changed it